backend/manager/modules/bll/src/test/java/org/ovirt/engine/core/bll/scheduling/commands/AddAffinityGroupCommandTest.java

```diff
@@ -64,11 +69,18 @@ public void setup() {
         VmStatic vmStatic = new VmStatic();
         vmStatic.setClusterId(clusterId);
         vmStatic.setId(vmId);
+        vmStatic.setName("vm");
         doReturn(Collections.singletonList(vmStatic)).when(vmStaticDao).getByIds(any());
 
+        VdsStatic vdsStatic = new VdsStatic();
+        vdsStatic.setName("host");
+        doReturn(Collections.singletonList(vdsStatic)).when(vdsStaticDao).getByIds(any());
```

Comment:
Also, I am not sure if this is necessary. We do mock affinityValidator, do we need to mock the call to vdsStaticDao too?